docs/design/2020-05-08-standardize-error-codes-and-messages.md

```diff
@@ -0,0 +1,166 @@
+# Proposal: Standardize error codes and messages
+
+- Author(s):     Joshua
+- Last updated:  May 8
+- Discussion at: https://docs.google.com/document/d/1beoa5xyuToboSx6e6J02tjLLX5SWzmqvqHuAPEk-I58/edit?usp=sharing
+
+## Abstract
+
+This proposal proposes to create a document to display error codes and error messages.
+
+## Motivation
+
+When certain errors occur in the TiDB system, an error message with an error code will be thrown. 
+In many cases, users don't know what the error message is saying. So we need a manual to locate 
+errors and solutions. In order to allow users to locate the cause of the error faster, we can 
+integrate the manual to the `doc` component of [TiUP](https://github.com/pingcap-incubator/tiup),
```

Comment:
A little weird that tiup occurs here, better to add an explanation of what is tiup.

Reply:
I have removed tiup from this document